io.github.mike632t.x11-calc.yaml

```diff
@@ -16,19 +17,18 @@ modules:
       - install -m755 bin/* /app/bin/
       - install -dm755 /app/share/x11-calc/prg
       - install -m644 prg/* /app/share/x11-calc/prg
-      - sed -i "s/^Icon=.*/Icon=$FLATPAK_ID/" src/x11-calc.desktop.linux
-      - install -Dm644 src/x11-calc.desktop.linux /app/share/applications/$FLATPAK_ID.desktop
+      - sed -i "s/^Icon=.*/Icon=$FLATPAK_ID/" x11-calc.desktop.linux
```

Comment:
this should be using `desktop-file-edit` instead of obscure sed.

Reply:
thanks for the suggestion: will come with next push (along with upstream changes)